Source/WebInspectorUI/UserInterface/Models/CSSGrouping.js

```diff
@@ -25,23 +25,48 @@
 
 WI.CSSGrouping = class CSSGrouping
 {
-    constructor(type, text, sourceCodeLocation)
+    constructor(nodeStyles, ownerStyleSheet, {id, type, text, sourceCodeLocation} = {})
     {
+        console.assert(nodeStyles);
         console.assert(Object.values(CSSGrouping.Type).includes(type));
```

Comment:
it seems like the `type` is required, so that should not be in the `options = {}`

Reply:
Oops, still need to do this.